tools/contractdeployer/README.md

```diff
@@ -6,7 +6,7 @@ Deploys contracts to both the obscuro network (L2) and eth network (L1)
 
 All commands are executed by running `contractdeployer/main/main()`.
 
-Contract is a string value to select the contract bytecode to deploy, currently only ERC20 and MGMT are supported
+Contract is a string value to select the contract bytecode to deploy, currently ERC20, MGMT and GUESS are supported
```

Comment:
Maybe worth replacing line 10 below this with documentation for what the different contracts do? I.e.
```
ERC20 - a standard ERC20 contract
MGMT - the Obscuro L1 management contract
GUESS - Guessing game demo app
```

Reply:
done